### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#199] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-27 134941](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/45473fb2-08eb-4b67-b506-fa28d7e0d499)

**Comment by cornflexxx** [comment, marked as answer]
## Soluzione confermata in classe
$1)$ Dato che: $$\int_{-\infty}^{+\infty} f(x)dx=1$$
$$\int_{0}^{k} (\frac{1}{6}x+k)dx=1$$
Svolgo l'integrale:
$$\frac{1}{12}k^2 + k^2=\frac{13}{12}k^2=1$$
Risolviamo l'equazione  per trovare $k$:

$$k=\sqrt{\frac{12}{13}}\\quad \text{Considero solo il $k>0$}$$

$2)$ $\mathbb{P}(1\leq X \leq 2)=0$
